### ISSUE TITLE
pipy testing

### ISSUE THREAD

**Issue by BeccaMallett** [Irwin-Blair/Group-F#11]
Write instructions for the user on how to install the package on Github.

**Comment by BeccaMallett** [comment]
I will set up an account with pypi.

**Comment by BeccaMallett** [comment]
pypi-AgENdGVzdC5weXBpLm9yZwIkNzRhOWY3NzgtNTMxYS00YzQ1LTk5NTUtOGJkZTcwZTViMmRlAAIqWzMsImJlNjdiZTAxLTMxMTUtNDI5ZS1hMjZmLTZmMzNhZDJlZWIxMSJdAAAGIFhLneUVb0hxfYmqgFgCAL-jPRf8cOr_buXjCQhXpEIZ

**Comment by BeccaMallett** [comment]
This is the token for pipy

**Comment by ciarms** [comment]
We can include the README file I've created.

**Comment by BeccaMallett** [comment]
that sounds good, still working on creating a package at the moment

**Comment by BeccaMallett** [comment]
@ciarms Once the README document is finished and updated, send it to me on this issue and I will add it into the package and upload once it has been spell checked

**Comment by BeccaMallett** [comment]
Can i use the function.py as the example of using the package? @Irwin-Blair

**Comment by BeccaMallett** [comment]
I have merged the branch and am creating a separate issue for the README.py for when it is complete then i will merge it with the main and upload the package
